main.py

```diff
@@ -51,7 +51,7 @@ def  divide(a, b):
 Tes Tes Test 2
 
 
-
+Dolev Dolev Dolev
```

Comment:
"Dolev Dolev Dolev" will not compile, add # before line

Reply:
Adding...